### PR TITLE
Inference pipeline

### PR DESCRIPTION
**Pull request by sdavrieux**
Check after merging train pipeline.

### CODE REVIEW

File: blurring_as_a_service/training_pipeline/components/train_model.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import os
```
Comment (epureanudiana) [inline blurring_as_a_service/training_pipeline/components/train_model.py:1]: why is this file pushed again? are there differences with the training pipeline PR?

Comment (epureanudiana) [reply blurring_as_a_service/training_pipeline/components/train_model.py:1]: same for the whole training_pipeline folder actually :D

File: blurring_as_a_service/inference_pipeline/components/blur_images.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+]
+
+
+@command_component(
+    name="blur_images",
+    display_name="Blur images.",
+    environment=f"azureml:{aml_experiment_settings['env_name']}:{aml_experiment_settings['env_version']}",
+    code="../../../",
+)
+def blur_images(
```
Comment (epureanudiana) [inline blurring_as_a_service/inference_pipeline/components/blur_images.py:29]: why is this in components and not in source? 
do we have some rule/indication when to split something into source and components and when to just use components?

Comment (sdavrieux) [reply blurring_as_a_service/inference_pipeline/components/blur_images.py:29]: Mmm some parts could be moved to source indeed, I will refactor it

File: blurring_as_a_service/inference_pipeline/components/blur_images.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                f"{results_path}/detection_result/{os.path.basename(image_to_blur)}",
+                image,
+            ):
+                raise Exception(
+                    f"Could not write image {os.path.basename(image_to_blur)}"
+                )
+        else:
+            shutil.copyfile(
+                f"{data_to_blur}/{image_to_blur}",
+                f"{results_path}/detection_result/{os.path.basename(image_to_blur)}",
```
[the remark this answers is not in the document]
Comment (sdavrieux) [reply blurring_as_a_service/inference_pipeline/components/blur_images.py:61]: Oh ok, I didn't know that. Let's keep both functionalities then and proper test it in the future

**Pull request merged**